lib/plugins/aws/invoke/index.js

```diff
@@ -119,6 +118,10 @@ class AwsInvoke {
       logResult.split('\n').forEach(line => this.consoleLog(formatLambdaLogEvent(line)));
     }
 
+    if (invocationReply.FunctionError) {
```

Comment:
Don't think so ... `InvocationReply` is returned from the AWS API and is irrelevant to invoke local I think